polaris/ocean/convergence/__init__.py

```diff
@@ -87,14 +87,13 @@ def get_timestep_for_task(config, refinement_factor,
     else:
         dt_per_km = section.getfloat('split_dt_per_km')
         btr_dt_per_km = section.getfloat('btr_dt_per_km')
-        for idx, refinement_factor in enumerate(refinement_factors):
```

Comment:
@cbegeman, I believe this for-loop is a bug so I have removed it. But I'd appreciate you having a look and verifying.